django-stubs/contrib/admin/helpers.pyi

```diff
@@ -170,7 +171,7 @@ class InlineAdminForm(AdminForm):
     def pk_field(self) -> AdminField: ...
     def fk_field(self) -> AdminField: ...
     def deletion_field(self) -> AdminField: ...
-    def ordering_field(self): ...
+    def ordering_field(self) -> AdminField: ...
```

Comment:
This is removed in Django upstream since code never used it, should be fine to remove it from stubs prematurely

https://github.com/django/django/commit/f3a06b06b59b3d24d5c2c14a34a8051879203c45

```suggestion
```